cockpit-starter-kit.spec.in

```diff
@@ -7,6 +7,8 @@ License: LGPLv2+
 Source: cockpit-starter-kit-%{version}.tar.gz
 BuildArch: noarch
 
+Requires: cockpit
```

Comment:
Let's please just depend on cockpit-system. Especially for remotely managed systems requiring cockpit-ws (through cockpit) is too much.